changelog/unreleased/changes/2453--readme-license-hyperlink.md

```diff
@@ -0,0 +1 @@
+Broken license hyperlink in README.md now points to valid LICENSE file in root of the repo.
```

Comment:
I don't think this is a user-facing change that deserves a changelog entry. Can you remove the file again?